libreosteoweb/templates/search/indexes/libreosteoweb/document_text.txt

```diff
@@ -1,3 +1,4 @@
 {{ object.title }}
 {{ object.date }}
-{{ object.notes }}
\ No newline at end of file
+{{ object.notes }}
+{{ object.job }}
```

Comment:
After reviewing I think it is not the right place to put this field. document_text is to index ... document for patient, and not for patient indexing.
Did  you test successfully this change ? (at least manually test ?)